src/main/views/opa-public-list.njk

```diff
@@ -0,0 +1,138 @@
+                <div class="govuk-warning-text">
+                    <span class="govuk-warning-text__icon align-warning-icon" aria-hidden="true">!</span>
+                    <strong class="govuk-warning-text__text">
+                        <span class="govuk-warning-text__assistive">Warning</span>
+                        {{ restrictionInformationBoldText }}
+                    </strong>
+                </div>
+                <p class="govuk-body">{{ restrictionInformationP2 }}</p>
+                <p class="govuk-body">{{ restrictionInformationP3 }}</p>
+                <p class="govuk-body">{{ restrictionInformationP4 }}</p>
+                <ul class="govuk-list govuk-list--bullet">
+                    <li>{{ restrictionBulletPoint1 }}</li>
+                    <li>{{ restrictionBulletPoint2 }}</li>
+                </ul>
+            </div>
+            {% if bill %}
+                <div class="govuk-body">
+                    <h3>{{ restrictionInformationHeadingCy }}</h3>
+                    <p class="govuk-body">{{ restrictionInformationP1Cy }}</p>
+                    <div class="govuk-warning-text">
+                        <span class="govuk-warning-text__icon align-warning-icon" aria-hidden="true">!</span>
+                        <strong class="govuk-warning-text__text">
+                            <span class="govuk-warning-text__assistive">Warning</span>
+                            {{ restrictionInformationBoldTextCy }}
+                        </strong>
+                    </div>
+                    <p class="govuk-body">{{ restrictionInformationP2Cy }}</p>
+                    <p class="govuk-body">{{ restrictionInformationP3Cy }}</p>
+                    <p class="govuk-body">{{ restrictionInformationP4Cy }}</p>
+                    <ul class="govuk-list govuk-list--bullet">
+                        <li>{{ restrictionBulletPoint1Cy }}</li>
+                        <li>{{ restrictionBulletPoint2Cy }}</li>
+                    </ul>
+                </div>
+            {% endif %}
+        </div>
+    </div>
+    <p class="govuk-body">{{ listDetail | replace("CASES_COUNT", length) }} {{ publishedDate }} {{ publishedAt }} {{ publishedTime }}</p>
+    {{ searchInput(text = searchCases) }}
+    <div class="govuk-grid-row">
+        <div class="parent-box govuk-!-margin-top-5 search-area">
+            <div class="overflow-table">
+                {% set opaCases = [] %}
+                {% for row in listData %}
+                    {% set _ = opaCases.push([
+                        {
+                            text: row.name
+                        },
+                        {
+                            text: row.caseUrn
+                        },
+                        {
+                            text: row.offence[0].offenceTitle + " - " + row.offence[0].offenceSection + "\n Reporting Restriction - " + row.offence[0].offenceReportingRestriction
```

Comment:
This doesn't look right. You only use the first array member. If there are multiple offences, the rest will not be displayed. Also I think the new line character will not work in nunjucks/HTML. You might want to try the <br> tag